.github/workflows/verify-benchmark-correctness.yml

```diff
@@ -0,0 +1,60 @@
+name: Verify Benchmarks
+
+on:
+  push:
+    branches: [develop, "release/*"]
+  pull_request:
+    branches: ["*"]
+
+env:
+  # Please ensure that this is in sync with graalVersion in build.sbt
+  graalVersion: 21.3.0
+  # Please ensure that this is in sync with javaVersion in build.sbt
+  javaVersion: 11
+  # Please ensure that this is in sync with project/build.properties
+  sbtVersion: 1.5.2
+
+jobs:
+  verify_benchmarks:
+    name: Verify Benchmarks
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    strategy:
+      # No need to run it on multiple distros, result should be the same
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup GraalVM Environment
+        uses: ayltai/setup-graalvm@v1
+        with:
+          graalvm-version: ${{ env.graalVersion }}
+          java-version: ${{ env.javaVersion }}
+          native-image: true
+      - name: Set Up SBT
+        shell: bash
+        run: |
+          curl --retry 4 --retry-connrefused -fsSL -o sbt.tgz https://github.com/sbt/sbt/releases/download/v${{env.sbtVersion}}/sbt-${{env.sbtVersion}}.tgz
+          tar -xzf sbt.tgz
+          echo $GITHUB_WORKSPACE/sbt/bin/ >> $GITHUB_PATH
+      - name: Setup conda
+        uses: s-weigand/setup-conda@v1.0.5
+        with:
+          update-conda: true
+          conda-channels: anaconda, conda-forge
+      - name: Install FlatBuffers Compiler
+        run: |
+          conda install flatbuffers=1.12.0
+          flatc --version
+          type flatc
+      - name: Build Engine
+        run: |
+          sbt "bootstrap"
+          sbt "buildEngineDistribution"
+      - name: Modify benchmarks to run just once
+        run: |
+          find engine/runtime/src/bench/java/org/enso/interpreter/bench/benchmarks/ -type f | xargs sed -i -e 's/@Warmup.*/@Warmup(iterations = 1)/' -e 's/@Measurement.*iterations.*/@Measurement(iterations = 1)/'
```

Comment:
```suggestion
          find engine/runtime/src/bench/java/org/enso/interpreter/bench/benchmarks/ -type f | xargs sed -i -e 's/@Warmup.*/@Warmup(iterations = 0)/' -e 's/@Measurement.*iterations.*/@Measurement(iterations = 1)/'
```

Reply:
To make the check even more quicker

Reply:
Used in [ac2b589](https://github.com/enso-org/enso/pull/3640/commits/ac2b5895f506264f78dfbf38084d8a8c25aa88c7), thanks.